internal/querynodev2/segments/segment_loader.go

```diff
@@ -1475,14 +1512,18 @@ func (loader *segmentLoader) LoadIndex(ctx context.Context,
 				return merr.WrapErrIndexNotFound("index file list empty")
 			}
 
-			fieldInfo, ok := fieldInfos[info.GetFieldID()]
-			if !ok {
-				return merr.WrapErrParameterInvalid("index info with corresponding field info", "missing field info", strconv.FormatInt(fieldInfo.GetFieldID(), 10))
-			}
-			err := loader.loadFieldIndex(ctx, segment, info)
-			if err != nil {
-				log.Warn("failed to load index for segment", zap.Error(err))
-				return err
+			if info.IsJson {
```

Comment:
Do we need to change the `getResourceUsageEstimateOfSegment` function per quantity and size of json indexes?

Reply:
> Do we need to change the `getResourceUsageEstimateOfSegment` function per quantity and size of json indexes?

we use inverted index to implement json indexes so we can reuse the memroy estimation of inverted index.